Depfiles/depinstall.sh

```diff
@@ -6,3 +6,8 @@ done < ./Depfiles/dependencies.pip
 while read p; do
 composer require p
 done < ./Depfiles/dependencies.comp
+
+if [“$HOSTNAME” = “venus”] || [“$HOSTNAME” = “mars”] ; then
```

Comment:
Did you test this?

Reply:
No. Is there a way to set the hostname? I have no idea what that line does, sorry

Reply:
`bash ./depfiles/depinstall.sh`

see what happens